### DISCUSSION TITLE
如何升级 Milvus 单机版本 v2.1.4 to v2.2.0

### DISCUSSION THREAD

**Discussion by bluestaree** [milvus-io/milvus#21287] [Q&A and General discussion] [answered]
按照官方文档步骤走： (https://milvus.io/docs/upgrade_milvus_standalone-docker.md)

在执行第三步命令时，控制台打印如下信息：

[2022/12/16 08:09:33.302 +00:00] [DEBUG] [sessionutil/session_util.go:185] ["Session try to connect to etcd"]
{"level":"warn","ts":"2022-12-16T08:09:38.307Z","logger":"etcd-client","caller":"v3@v3.5.0/retry_interceptor.go:62","msg":"retrying of unary invoker failed","target":"etcd-endpoints://0xc0002e81c0/#initially=[milvus-etcd:2379]","attempt":0,"error":"rpc error: code = DeadlineExceeded desc = latest balancer error: last connection error: connection error: desc = \"transport: Error while dialing dial tcp: lookup milvus-etcd on 127.0.0.11:53: no such host\""}
[2022/12/16 08:09:38.307 +00:00] [DEBUG] [retry/retry.go:39] ["retry func failed"] ["retry time"=20] [error="context deadline exceeded"]
[2022/12/16 08:09:41.309 +00:00] [DEBUG] [sessionutil/session_util.go:185] ["Session try to connect to etcd"]
{"level":"warn","ts":"2022-12-16T08:09:46.310Z","logger":"etcd-client","caller":"v3@v3.5.0/retry_interceptor.go:62","msg":"retrying of unary invoker failed","target":"etcd-endpoints://0xc0002e81c0/#initially=[milvus-etcd:2379]","attempt":0,"error":"rpc error: code = DeadlineExceeded desc = latest balancer error: last connection error: connection error: desc = \"transport: Error while dialing dial tcp: lookup milvus-etcd on 127.0.0.11:53: no such host\""}

是否需要其他配置？

**Comment by yhmo** [comment, marked as answer]
访问不到etcd了，那个migration需要访问etcd去更新里面的内容，docker ps看一下etcd还在否，如果etcd还在运行，那么检查一下ip/port对不对得上

**Reply by bluestaree** [reply]
好的 ,非常感谢

**Reply by youzhi123** [reply]
请问题主解决了吗

**Comment by youzhi123** [comment]
我也遇到这个问题了，请问题主怎么解决的

**Reply by bluestaree** [reply]
已解决，按官网文档步骤走就好 
https://milvus.io/docs/v2.2.x/upgrade_milvus_standalone-docker.md
